### ISSUE TITLE
Cockatrice armour

### ISSUE THREAD

**Issue by UCoala** [AlexModGuy/Ice-and-Fire-Suggestions#927]
i just love this mob, i think will be cool put armour in then just like the hippogriff

**Comment by CLittleLunaC** [comment]
I agree, They work well as guards for minecolonies having armor for them would be awesome
